.github/workflows/ci.yml

```diff
@@ -20,8 +20,16 @@ jobs:
         os:
           - macos-latest
           - windows-latest
+          - ubuntu-latest
         python-version:
           - 2.7
+          - 3.6
+        exclude:
+          # Do not run coverage tests with Python 3.6 on Windows for
+          # now.  They will fail.  Dealing with them separately would
```

Comment:
Sounds like a good thing for which to have a ticket?  Then this comment could link there and folks could discover more details (or fix it).

Reply:
Filed https://tahoe-lafs.org/trac/tahoe-lafs/ticket/3669 to track that, and updated the comment.